easybuild/easyconfigs/c/Cube/Cube-4.3-foss-2015a.eb

```diff
@@ -0,0 +1,38 @@
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+# Copyright:: Copyright 2013 Juelich Supercomputing Centre, Germany
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+# License::   New BSD
+#
+# This work is based from experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+easyblock = 'EB_Score_minus_P'
+
+name = "Cube"
+version = "4.3"
+
+homepage = 'http://www.scalasca.org/software/cube-4.x/download.html'
+description = """Cube, which is used as performance report explorer for Scalasca and
+ Score-P, is a generic tool for displaying a multi-dimensional performance space
+ consisting of the dimensions (i) performance metric, (ii) call path, and (iii) system
+ resource. Each dimension can be represented as a tree, where non-leaf nodes of the tree
+ can be collapsed or expanded to achieve the desired level of granularity."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://apps.fz-juelich.de/scalasca/releases/cube/%(version)s/dist']
+
+
+# patches = ['Cube-4.2_fix-Qt-version-check.patch']
+
+dependencies = [('Qt', '4.8.6')]
+
+configopts = "QMAKE=$EBROOTQT/bin/qmake MOC=$EBROOTQT/bin/moc UIC=$EBROOTQT/bin/uic"
```

Comment:
It's actually a one-liner. Therefore I decided to include it in https://github.com/hpcugent/easybuild-easyblocks/pull/552 rather than opening a separate PR.